client/src/telethon/_impl/client/client/client.py

```diff
@@ -216,6 +216,7 @@ def __init__(
         datacenter: Optional[DataCenter] = None,
         connector: Optional[Connector] = None,
     ) -> None:
+        assert isinstance(__package__, str)
```

Comment:
Huh?

Reply:
`__package__` type is `Optional[str]`. The language server is showing an error with __package__ as None